packages/react-examples/src/react/DetailsList/DetailsList.Documents.Example.tsx

```diff
@@ -94,9 +95,11 @@ export class DetailsListDocumentsExample extends React.Component<{}, IDetailsLis
         minWidth: 16,
         maxWidth: 16,
         onColumnClick: this._onColumnClick,
-        onRender: (item: IDocument) => {
-          return <img src={item.iconName} className={classNames.fileIconImg} alt={item.fileType + ' file icon'} />;
-        },
+        onRender: (item: IDocument) => (
+          <TooltipHost content={`${item.fileType} file`} id={`file-type-tooltip-${item.key}`}>
```

Comment:
@JustSlone Curious why you suggested using TooltipHost here rather than just adding `title` on the `img`? (this was in a comment on the linked issue)

Reply:
@ecraig12345 In general we've been seeing more and more issues caused by the use of html title. That's why I suggested using a tooltip here instead. In this case title probably could work, but I still think we should prefer tooltip. @smhigley might have additional input.

The issues we see with tooltip are:
1. Html title doesn't display on keyboard focus (not an issue in this specific case, but if we update to support cell based navigation it probably would become an issue).
2. We also have been seeing a number of double announce bugs where both the content and title get announced

Lastly there is the point on consistency, if we have to use rich tooltips due to the above or other issues, it's a bit strange to also have html tooltips sprinkled throughout the application. 

So, in this case title would be enough to satisfy the bug, but I would suggest preferring TooltipHost. 🤷 

We should probably have a discussion to decide on an official stance at some point (i.e. should we work wholesale to move away from the title attribute).

Reply:
My personal preference would be to have no tooltips at all, and add a legend somewhere instead (if this were a real website, but maybe even for an example) 😄

Reply:
That all makes sense. I think the main issue I've seen with using TooltipHost not title is that if it's used *inside* a component which wouldn't otherwise reference tooltip/callout/layer/popup, that can cause major bundle size issues. But in an example that's not an issue.

The legend is an interesting idea, but I suspect tooltips would more closely resemble the way a document list would actually be implemented. Though I went to look at how it's done in SharePoint/OneDrive for business and was surprised to find that they have no tooltip or title for the icons at all. OneDrive consumer appears to use `title`.